code/game/objects/items/emags.dm

```diff
@@ -73,6 +88,22 @@
 			return FALSE
 	return TRUE
 
+/// What happens if a crewmember uses this card
+/obj/item/card/emag/proc/crew_action(mob/user)
+	if(!HAS_TRAIT(user, TRAIT_EMAG_USER))
+		crew_uses += 1
+	if(!prob(crew_uses * 25))
+		return
+	to_chat(user, span_warning("You hear a buzzing from [src] 'Non-authorised personel detected. Terminating card'."))
```

Comment:
needs to add a flag to prevent the emag from just being used until it actually goes boom